changelog.txt

```diff
@@ -8,6 +8,7 @@
 * Fix - On HPOS stores, when a subscription is loaded from the database, make sure all core subscription properties are read directly from meta.
 * Fix - When viewing My Account > Subscriptions, fix an issue where no subscriptions were listed when HPOS is enabled.
 * Fix - On HPOS stores, ensure payment tokens are copied from the subscription to the renewal order.
+* Fix - Refactor `WCS_Meta_Box_Subscription_Data::save` to support HPOS stores, fixing a PHP warning notice when updating an order via the Edit Order screen.
```

Comment:
Unsure whether to keep 2 similar changelog entries separate (line 11 + line 12) or merge into one.

Reply:
I think it's OK to keep 2 similar changelog entries.